packages/core/src/alert/alert-group.stories.ts

```diff
@@ -273,6 +420,93 @@ export function bannerGroup() {
   `;
 }
 
+export const darkThemeBannerAlertGroup = () => {
+  return html`
+    <div cds-theme="dark" cds-layout="vertical gap:sm">
+      <cds-alert-group type="banner" status="info">
+        <cds-alert closable>
+          This example is a closable banner alert inside a banner alert group with a status of "info".
+        </cds-alert>
+      </cds-alert-group>
+
+      <cds-alert-group type="banner" status="success">
+        <cds-alert closable>
+          This example is a closable alert with a status of "success" inside a banner alert group.
+        </cds-alert>
+      </cds-alert-group>
+
+      <cds-alert-group type="banner" status="warning">
+        <cds-alert closable>
+          This example is a closable alert with alert action buttons and a status of "warning" inside a banner alert
+          group.
+          <cds-alert-actions>
+            <cds-button>Button 1</cds-button>
+            <a href="#">
+              <cds-button>Link 1</cds-button>
+            </a>
+          </cds-alert-actions>
+        </cds-alert>
+      </cds-alert-group>
+
+      <cds-alert-group type="banner" status="danger">
+        <cds-alert closable>
+          This example is a closable alert with a status of "danger" inside a banner alert group.
+          <cds-alert-actions>
+            <cds-button>Button 1</cds-button>
+            <a href="#">
+              <cds-button>Link 1</cds-button>
+            </a>
+          </cds-alert-actions>
+        </cds-alert>
+      </cds-alert-group>
+
+      <cds-alert-group type="banner" status="neutral">
+        <cds-alert closable>
+          This example is a closable alert with a status of "neutral" inside a banner alert group.
+          <cds-alert-actions>
+            <cds-button>Button 1</cds-button>
+            <a href="#">
+              <cds-button>Link 1</cds-button>
+            </a>
+          </cds-alert-actions>
+        </cds-alert>
+      </cds-alert-group>
+    </div>
+  `;
+};
+
+export const legacyWarningAlertBanner = () => {
```

Comment:
Demo showing how to override css to make warning alert match the exact color of clr-angular